storage/byodb/external-buckets/index.md

```diff
@@ -61,8 +61,8 @@ visible in Keboola.
 {: .image-popup}
 ![Bucket refresh](/storage/byodb/external-buckets/figures/4.png)
 
-External datasets cannot be used in an [input mapping](/transformations/mappings/#input-mapping)
-They are accessible via the [read-only input mapping](/transformations/mappings/#read-only-input-mapping). 
+External datasets can be used in an [input mapping](/transformations/mappings/#input-mapping)
```

Comment:
@jirkasemmler myslíš že to stačí takto, nebo to mám nějak víc celé popsat?

Reply:
za me to urcite staci, ale tady bych to od tohoto bodu rad nechal na @tom-hegedus protoze to zavisi a asi dlouhodobe bude i zaviset na te nove ficure - jestli ji odstranime nebo ne, jestli to zablokujeme i na backendu, protoze costy apod...